### PR TITLE
Point to new Web3JS docs site

### PR DESCRIPTION
**Pull request by mlibre**
update web3js

Update link to web3js. from "https://web3js.readthedocs.io/en/v1.8.1/" to "https://docs.web3js.org/"

### CODE REVIEW

File: src/content/developers/docs/consensus-mechanisms/pos/index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Whereas under proof-of-work, the timing of blocks is determined by the mining di
...
+1. A user creates and signs a [transaction](/developers/docs/transactions/) with their private key. This is usually handled by a wallet or a library such as [ether.js](https://docs.ethers.io/v5/), [web3js](https://docs.web3js.org/), [web3py](https://web3py.readthedocs.io/en/v5/) etc but under the hood the user is making a request to a node using the Ethereum [JSON-RPC API](/developers/docs/apis/json-rpc/). The user defines the amount of gas that they are prepared to pay as a tip to a validator to encourage them to include the transaction in a block. The [tips](/developers/docs/gas/#priority-fee) get paid to the validator while the [base fee](/developers/docs/gas/#base-fee) gets burned.
```
Comment (minimalsm) [inline src/content/developers/docs/consensus-mechanisms/pos/index.md:36]: I'm a little confused by the Web3JS docs.

We have:

- https://web3js.readthedocs.io/en/ (which redirects to v1.10.0)
- We also have v.3.0.0 in the dropdown of the RTD version of the docs
- And we have v4.0.x at https://docs.web3js.org/

Not really sure where we should be directing people...

I've asked in the ChainSafe Discord for clarity here: https://discord.com/channels/593655374469660673/719602820801888337/1128248081289326632

Comment (minimalsm) [reply src/content/developers/docs/consensus-mechanisms/pos/index.md:36]: Either way, we probably shouldn't be hardcoding versions (v1.8.1) unless we have a good reason to...

Thanks for raising this @mlibre ❤️

Comment (corwintines) [reply src/content/developers/docs/consensus-mechanisms/pos/index.md:36]: Updated based on https://discord.com/channels/593655374469660673/719602820801888337/1128359644742557717

**Review by corwintines — APPROVED** [review]
Thanks @mlibre

**Pull request merged**